features/view_specs/view_spec.feature

```diff
@@ -212,3 +212,24 @@ Feature: view spec
       """
     When I run `rspec spec/views`
     Then the examples should all pass
+
+  Scenario: path helpers should take precedence over asset helper
```

Comment:
> Yes but consider the case where someone is testing that an output matches an asset path

In that case, that said someone would already have to use [`path_to_video`](http://api.rubyonrails.org/classes/ActionView/Helpers/AssetUrlHelper.html#method-i-path_to_video) method in their view becasue if they're calling `video_path` in their view file (while also having a route name `video`) it'll actually call the route helper instead of the asset helper. Otherwise, they are probably testing the wrong thing.

So, to recap: with `resources :videos` in your route file

* Behavior before this patch:
    * `video_path` in view -> route helper
    * `video_path` in view spec -> asset helper
* Behavior after this patch:
    * `video_path` in view -> route helper
    * `video_path` in view spec -> route helper

--- 

> if you absolutely need to run a spec to test this then you need to check the run_count and check for failures

Aha! So that's what it was for. I'll rewrite my test then.